test/distributed/org/apache/cassandra/distributed/test/FailingRepairTest.java

```diff
@@ -167,7 +168,14 @@ public static void teardownCluster() throws Exception
     public void cleanupState()
     {
         for (int i = 1; i <= CLUSTER.size(); i++)
-            CLUSTER.get(i).runOnInstance(InstanceKiller::clear);
+            CLUSTER.get(i).runOnInstance(() -> {
+                InstanceKiller.clear();
```

Comment:
that's probably fine, but I'd like to mention that on `trunk` it's been solved differently, though the main idea looks similar